### ISSUE TITLE
AttributeError: 'Matches' object has no attribute 'as_type'

### ISSUE THREAD

**Issue by frigusgulo** [ezwelty/glimpse#33]
Error: 
 ```
  '
    File "/home/dunbar/anaconda3/envs/glimpse/lib/python3.8/site-packages/glimpse/optimize.py",
 line 2567, in process
      match = match.as_type(as_type)
  AttributeError: 'Matches' object has no attribute 'as_type'
  
  '
  ```

Code:

`

				matcher.build_keypoints(
    				clear_images=True,
        			overwrite=True,
        			masks=mask)

				matcher.build_matches(
        			maxdt=MAXDT, 
        			seq=MATCH_SEQ,
        			path=TOUNGE_MATCHES_PATH,
			        max_ratio=0.75,
			        weights=True,
			        as_type=glimpse.optimize.RotationMatchesXY
			        )

`

**Comment by ezwelty** [comment]
Thanks for catching. The sin of an incomplete error suite.
Closed by #34.

**Comment by frigusgulo** [comment]
Roger.
